util/setup_base.txt

```diff
@@ -40,60 +42,49 @@ const config = {
     // This is your permission level, the staff levels should always be above the rest of the roles.
     { level: 2,
       // This is the name of the role.
-      name: "Moderator",
+      name: 'Trainee',
       // The following lines check the guild the message came from for the roles.
       // Then it checks if the member that authored the message has the role.
       // If they do return true, which will allow them to execute the command in question.
       // If they don't then return false, which will prevent them from executing the command.
       check: (message) => {
         try {
-          const modRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.modRole.toLowerCase());
-          if (modRole && message.member.roles.has(modRole.id)) return true;
+          const traineeRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.traineeRole.toLowerCase());
+          if (traineeRole && message.member.roles.has(traineeRole.id)) return true;
         } catch (e) {
           return false;
         }
       }
     },
 
     { level: 3,
-      name: "Administrator", 
+      name: 'Council', 
       check: (message) => {
         try {
-          const adminRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.adminRole.toLowerCase());
-          return (adminRole && message.member.roles.has(adminRole.id));
+          const councilRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.councilRole.toLowerCase());
+          return (councilRole && message.member.roles.has(councilRole.id));
```

Comment:
If you don't feel like going through with Ian's suggestion, this entire block can be replaced with:

```js
message.member.roles
  .find(role => role.name.toLowerCase(), message.settings.councilRole.toLowerCase())
```